### PR TITLE
(7) feat: add disclaimer screen

### PR DESCRIPTION
**Pull request by alexruzenhack**
### Acceptance Criteria
- Add disclaimner screen

>[!NOTE]
>I edited some commits to remove module aliases.

https://github.com/HathorNetwork/hathor-wallet-mobile/assets/5992210/037dc923-e64e-4078-9e96-7d8911f5a242

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

**Review by andreabadesso — CHANGES_REQUESTED** [review]

**Review by pedroferreira1 — COMMENTED** [review]
I think I prefer the text to be more clear as a warning. I don't like having lot of text in yellow but maybe we could have a box with yellow background and this same text written inside the box in white or black. What do you think? Can you try something like that?

File: src/screens/NetworkSettings/NetworkSettingsDisclaimerScreen.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (alexruzenhack) [reply src/screens/NetworkSettings/NetworkSettingsDisclaimerScreen.js:7]: - Add copyright: 9a70ad4cdb85c5a1ee519174d3a65cdbb417cf92

**Comment by alexruzenhack** [comment]
> I think I prefer the text to be more clear as a warning. I don't like having lot of text in yellow but maybe we could have a box with yellow background and this same text written inside the box in white or black. What do you think? Can you try something like that?

See if now it is better...

File: src/screens/NetworkSettings/NetworkSettingsDisclaimerScreen.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  },
+  centeredText: {
+    fontSize: 16,
+  },
+  buttonContainer: {
+    alignSelf: 'stretch',
+  },
+});
+
+export const NetworkSettingsDislaimerNav = Symbol('NetworkSettingsDislaimer').toString();
```
[the remark this answers is not in the document]
Comment (alexruzenhack) [reply src/screens/NetworkSettings/NetworkSettingsDisclaimerScreen.js]: - Fix typo on disclaimer names: 1078ed2302e99e0d406c104a503e3f84141b1933

File: src/screens/NetworkSettings/NetworkSettingsDisclaimerScreen.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+export function NetworkSettingsDislaimerScreen({ navigation }) {
```
[the remark this answers is not in the document]
Comment (alexruzenhack) [reply src/screens/NetworkSettings/NetworkSettingsDisclaimerScreen.js]: - Fix typo on disclaimer names: 1078ed2302e99e0d406c104a503e3f84141b1933

File: src/screens/NetworkSettings/NetworkSettingsDisclaimerScreen.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  return (
+    <View style={style.container}>
+      <HathorHeader
+        title={riskDisclaimerTitleText}
+        onBackPress={() => navigation.goBack()}
+      />
+      <View style={style.content}>
+        <View>
+          <Text style={style.centeredText}>
+            {t`Do not change this unless you know what your are doing, and under no circustance change this settings based on someone else suggestion, as this can potentially make you susceptible to fraudulent schemes.`}
```
[the remark this answers is not in the document]
Comment (alexruzenhack) [reply src/screens/NetworkSettings/NetworkSettingsDisclaimerScreen.js]: - Fix typo on message: a193f82fd68984f3745130905184011584bcad03

**Review by pedroferreira1 — APPROVED** [review]

**Review by andreabadesso — APPROVED** [review]

**Pull request merged**